odpf/stencil/v1beta1/stencil.proto

```diff
@@ -295,3 +305,21 @@ message DeleteVersionRequest {
 message DeleteVersionResponse {
   string message = 1;
 }
+
+message SearchSchemasRequest {
+  string namespace_id = 1;
+  int32 version_id = 2;
```

Comment:
@ramey Version alone will not make sense here, right? The stencil will have many schemes and each schema will have different versions. So we if want to search specifically then we will need schema id as well. 
IMO, Specificity of search will work in following manner:   namespace > schema > version

Reply:
Makes sense @ravisuhag, will remove version.
In that case shall we also support  `/namespace/{namespace_id}/schemas/{schema_id}/_search` and `/namespace/{namespace_id}/schemas/{schema_id}/versions/{version_id}/_search` APIs?

Reply:
@ramey I think we should support a `/search` API which by default will search across all namespaces, schemas, and versions(latest by default). And it will also take query params for namespace, schema, and version to filter the result down. WDYT?

Reply:
Done changes as discussed.